src/screens/wallet/SendToScreen.tsx

```diff
@@ -120,6 +122,9 @@ const SendToScreen = (props: any) => {
     }
   }, [amountInString, currentAmount]);
 
+  const sendTransactionPaddingBottom = {
```

Comment:
should this be refactored out?

Reply:
Yes, What do you think can be done instead?

Reply:
I meant as part of `sendToScreenStyles`